lib/shark_on_lambda/configuration.rb

```diff
@@ -74,6 +79,12 @@ def root=(new_root)
       @root = Pathname.new(new_root)
     end
 
+    def routes
+      @routes ||= ActionDispatch::Routing::RouteSet.new_with_config(
+        ActionDispatch::Routing::RouteSet::Config.new(nil, true)
```

Comment:
Meaning 
```
irb(main):001:0> ActionDispatch::Routing::RouteSet::Config.new(nil, true)
=> #<struct ActionDispatch::Routing::RouteSet::Config relative_url_root=nil, api_only=true>
```
Do we want an explanation for `nil` and `true` here?

Reply:
Actually, I have to try out if `relative_url_root = 'form-service'` helps us out with shorter paths.